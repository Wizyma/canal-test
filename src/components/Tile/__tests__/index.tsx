import { render, screen } from '@testing-library/react';

import Component from '..'

test(`It should render the ${Component.name} component`, () => {
  const { asFragment } = render(<Component title="Jest Test" poster_path='/nothing' />)

  // ensure element has been rendered
  expect(screen.getByText('Jest Test')).toHaveTextContent('Jest Test');
  expect(asFragment()).toMatchSnapshot()
})