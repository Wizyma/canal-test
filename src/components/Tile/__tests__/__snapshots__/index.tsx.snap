// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render the Tile component 1`] = `
<DocumentFragment>
  <div
    class="jsx-2747747579 card "
  >
    <div
      class="jsx-363961758 image "
    >
      <div
        class="jsx-2292771821 skeleton"
      />
      <img
        class="jsx-363961758 "
        height="600"
        src="https://image.tmdb.org/t/p/w500/nothing"
        style="object-fit: cover;"
        width="400"
      />
    </div>
    <div
      class="jsx-2956695818 content "
      style="width: auto;"
    >
      <p
        class="jsx-3390086683  "
        style="text-align: center; font-weight: 700; max-width: 350px;"
      >
        Jest Test
      </p>
    </div>
  </div>
</DocumentFragment>
`;
